docsets/Poco/docset.json

```diff
@@ -1,14 +1,18 @@
 {
     "name": "Poco",
-    "version": "1.9.0",
+    "version": "1.9.4",
     "archive": "Poco.tgz",
     "author": {
         "name": "wxk6b1203",
         "link": "https://github.com/wxk6b1203"
     },
     "aliases": ["poco", "pocoproject"],
     "specific_versions": [{
-         "version": "1.9.0",
-         "archive": "versions/1.9.0/Poco.tgz",
-    }]
+         "version": "1.9.4",
+         "archive": "versions/1.9.4/Poco.tgz"
```

Comment:
You didn't add an archive at this path. I think you added it at `versions/Poco.tgz` instead.